tests/functional/health.ts

```diff
@@ -0,0 +1,57 @@
+/* tslint:disable:no-relative-imports */
+import { describe } from 'riteway'
+import { app } from '../../src/app'
+import { getHealth } from '../helpers/health'
+import { delay, runtimeId, createDatabase } from '../helpers/utils'
+
+const PREFIX = `test-functional-node-poet-${runtimeId()}`
+const NODE_PORT = '28081'
+
+const blockchainSettings = {
+  MINIMUM_BLOCK_HEIGHT: 100,
+  ENABLE_TIMESTAMPING: true,
+  TIMESTAMP_INTERVAL_IN_SECONDS: 10,
+  BATCH_CREATION_INTERVAL_IN_SECONDS: 5,
+  READ_DIRECTORY_INTERVAL_IN_SECONDS: 5,
+}
+
+describe('Health Endpoint Returns the Correct Fields', async (assert: any) => {
+  const db = await createDatabase(PREFIX)
+  const server = await app({
+    BITCOIN_URL: process.env.BITCOIN_URL || 'bitcoind-1',
+    API_PORT: NODE_PORT,
+    MONGODB_DATABASE: db.settings.tempDbName,
+    MONGODB_USER: db.settings.tempDbUser,
+    MONGODB_PASSWORD: db.settings.tempDbPassword,
+    EXCHANGE_PREFIX: PREFIX,
+    ...blockchainSettings,
```

Comment:
`blockchainSettings` can be removed here and above. The default values are ok since this test is not doing anything to checking claims, etc. so timing is not an issue.